### PR TITLE
auto OCP-24340

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  could you help to review? Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/operator_hub.xyaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+create_custom_resource:
+  params:
+    button_text: Create
+  action: click_provided_apis_of_operators
+  action: click_button
+click_provided_apis_of_operators:
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.3/operator_hub.xyaml]: Can this rule be replaced by click_tab?

Comment (yanpzhan) [reply lib/rules/web/admin_console/4.3/operator_hub.xyaml]: I checked the rule with wrong page. Now it's reasonable from the expected page.

**Comment by SSshahan** [comment]
Thanks all, @akostadinov  @pruan-rht  could you help to merge it? thanks!

**Comment by SSshahan** [comment]
@akostadinov @pruan-rht  Since Its feature file has been already merged, then case failed with no relevant rules in testrun. Could you help to merge it, thanks！

**Pull request merged**